lib/dishes.rb

```diff
@@ -0,0 +1,14 @@
+class Dishes
+
+  def initialize
+    @list = [ "fish £6",
+            "chicken £7",
+             "curry £5",
+             "pizza £4",
+             "vegetarian £3"]
```

Comment:
Good use of an array. Maybe a hash with key and value pairs for later tests when refactoring, however I am not completely sure myself.